src/tools/web/elektrad/http_test.go

```diff
@@ -0,0 +1,92 @@
+package main_test
+
+// func TestKdbGet(t *testing.T) {
+// 	compareGet(t, "/kdb/system/elektra/modules/dump")
```

Comment:
commented out?

Reply:
Yes - these were the first tests that compared the output of the old and new elektrad. Since v1 is deleted I will remove these tests

Reply:
It would be good if you keep similar tests with static output comparison as regression tests.